### ISSUE TITLE
Improve labelling of list view filter

### ISSUE THREAD

**Issue by seanredmond** [nypl-spacetime/maps-by-decade#91]
"Filter by decade:" and "Filter by title:" aren't read by screenreaders. "Filter by decade:" should probably be a `<legend>` of the radio buttons `<fieldset>` and "Filter by title:" the `<legend>` of the text box.

Also, placeholder text should be used for an example of what can go in the field, not a sort of label
